src/_data/team.yml

```diff
@@ -859,4 +859,17 @@
   linkedin: jmir17
   email: josep.mir@codurance.com
   office: barcelona
-  
\ No newline at end of file
+
+- id: gonzalo_gomez_sullain
+  name: Gonzalo Gómez Sullain
+  image: /assets/custom/img/team/gonzalo_gomez_sullain.png
+  role: Software Craftsperson
+  fullDescription:
+    <p>Gonzalo joined Codurance after spending more than a decade actively in the software industry in diverse domains such as the army, healthcare, oil & gas and telematics systems.</p>
+    <p>Having started developing at his early age in Argentina, continued his career in electronics and later graduated in information systems engineering. While even studying, he got involved in teaching programming paradigms in the university.</p>
+    <p>Mostly involved in development and software engineering, he has special interest in behavioural and motivational aspects of high skilled teams, thus he has an Agile Coaching background.</p>
+    <p>Tries to travel as much as possible and enjoys sharing his experiences in a blog. Loves playing football and going to matches (no matter the division!). Swimming is a way he finds to renew energies. Enjoys techno music and is always trying to speak different lenguages.</p>
```

Comment:
```suggestion
    <p>He tries to travel as much as possible and enjoys sharing his experiences in a blog. He loves playing football and going to matches (no matter the division!). Swimming is a way he finds to renew energies. He enjoys listening to techno music. And he is always trying to speak different languages.</p>
```

Reply:
Not sure about 

> Swimming is a way he finds to renew energies

I need to think of a better alternative.

How about adding a link to your blog in your bio?

Reply:
Perhaps:

> He swims to rejuvenate

Its better, but it could be better. Lets ask @emmabeans if she can suggest a better alternative.

Apart from that one sentence, everything else seems good 👍

Reply:
I have a couple of suggestions - feel free to use/omit as desired! Apologies if the formatting isn't standard 'Git Hub' - I'm learning! 

After the sentence ending in 'telematics systems', I'd suggest: 

"Having started developing at an early age in Argentina, he continued his career in electronics before graduating in Information Systems Engineering - he was even involved in teaching programming paradigms at the university while studying!". 

"Gonzalo has a special interest in the behavioural and motivational aspects of highly skilled development and software engineering teams and has an Agile Coaching background. He tries to travel as much as possible and enjoys sharing his experiences via his blog. 

In his spare time, he is a fan of playing and attending football matches (no matter the team or division!) and is a fan of a rejuvenating swim. He can otherwise be found listening to techno music and trying to learn other languages."

Reply:
Thank you all for the suggestions. I´ve added the @emmabeans version, with some subtle change, as the log link (not sure about this, it is in Spanish...)

Let´s try.